### PR TITLE
Add specifier for message

### PR DESCRIPTION
**Pull request by iowillhoit**
### What does this PR do?
We will be changing how `util.format` works when formatting messages. 
Specifiers (eg `%s`) will be required for the data to be rendered. As is, it appends data if no matching specifier is found.

### What issues does this PR fix or reference?
More info here: https://github.com/forcedotcom/sfdx-core/pull/1099
[@W-16197665@](https://gus.my.salesforce.com/apex/ADM_WorkLocator?bugorworknumber=W-16197665)

### CODE REVIEW

File: messages/package_delete.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ The request to delete this package was canceled
 
 # humanSuccess
 
-Successfully deleted the package.
+Successfully deleted the package. %s
```
Comment (jshackell-sfdc) [inline messages/package_delete.md:51]: Do you need to move the period?  i.e. "Successfully delete the package %s. "

File: messages/package_version_delete.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ The request to delete this package version has been canceled.
 
 # humanSuccess
 
-Successfully deleted the package version.
+Successfully deleted the package version. %s
```
Comment (jshackell-sfdc) [inline messages/package_version_delete.md:49]: Ditto

File: messages/package_delete.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ The request to delete this package was canceled
 
 # humanSuccess
 
-Successfully deleted the package.
+Successfully deleted the package with id: %s
```
Comment (jshackell-sfdc) [inline messages/package_delete.md:51]:
```suggestion
Successfully deleted the package with ID: %s
```

File: messages/package_version_delete.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ The request to delete this package version has been canceled.
 
 # humanSuccess
 
-Successfully deleted the package version.
+Successfully deleted the package version with id: %s
```
Comment (jshackell-sfdc) [inline messages/package_version_delete.md:49]:
```suggestion
Successfully deleted the package version with ID: %s
```

File: test/commands/package/packageCreateAndDelete.nut.ts [hunk 5]
```diff
@@ -51,7 +51,7 @@ describe('package create/update/delete', () => {
     it('should delete a package - human readable results', () => {
       const command = `package:delete -p ${pkgName} -v ${session.hubOrg.username} --no-prompt`;
       const output = execCmd(command, { ensureExitCode: 0 }).shellOutput.stdout;
-      expect(output).to.contain('Successfully deleted the package. 0Ho');
+      expect(output).to.contain('Successfully deleted the package with id: 0Ho');
```
Comment (jshackell-sfdc) [inline test/commands/package/packageCreateAndDelete.nut.ts:54]:
```suggestion
      expect(output).to.contain('Successfully deleted the package with ID: 0Ho');
```

File: test/commands/package/packageVersion.nut.ts [hunk 6]
```diff
@@ -443,7 +443,7 @@ describe('package:version:*', () => {
       const id = deletableVersionIds.pop();
       const command = `package:version:delete -p ${id} --noprompt`;
       const result = execCmd<[PackageSaveResult]>(command, { ensureExitCode: 0 }).shellOutput.stdout;
-      expect(result).to.contain(`Successfully deleted the package version. ${id}`);
+      expect(result).to.contain(`Successfully deleted the package version with id: ${id}`);
```
Comment (jshackell-sfdc) [inline test/commands/package/packageVersion.nut.ts:446]:
```suggestion
      expect(result).to.contain(`Successfully deleted the package version with ID: ${id}`);
```

File: test/commands/package/version.delete.test.ts [hunk 7]
```diff
@@ -93,7 +93,7 @@ describe('package:version:delete', () => {
     const results: PackageSaveResult = await command.run();
     expect(results.id).to.equal('testId');
     expect(uxSuccessStub.calledOnce).to.be.true;
-    expect(uxSuccessStub.args[0][0]).to.equal('Successfully deleted the package version. testId');
+    expect(uxSuccessStub.args[0][0]).to.equal('Successfully deleted the package version with id: testId');
```
Comment (jshackell-sfdc) [inline test/commands/package/version.delete.test.ts:96]:
```suggestion
    expect(uxSuccessStub.args[0][0]).to.equal('Successfully deleted the package version with ID: testId');
```

**Review by jshackell-sfdc — APPROVED** [review]
See my edits

**Review by mdonnalley — APPROVED** [review]

**Pull request merged**